src/merge.py

```diff
@@ -0,0 +1,52 @@
+"""Merge sort module."""
+
+
+def merge_sort(seq):
+    """Sort sequence by merge sort."""
+    if isinstance(seq, list):
+        mid = len(seq) // 2
+        left, right = seq[:mid], seq[mid:]
+        if len(left) > 1:
+            left = merge_sort(left)
+        if len(right) > 1:
+            right = merge_sort(right)
+        sorted_seq = []
+        while left and right:
+            if left[-1] >= right[-1]:
+                sorted_seq.insert(0, left.pop())
+            else:
+                sorted_seq.insert(0, right.pop())
+        return list(left or right) + sorted_seq
+    else:
+        raise TypeError('Input type must be a list.')
+
+
+if __name__ == '__main__':
+    import timeit as ti
+    list1 = [1, 2, 4, 6]
+    list2 = [9, 8, 7, 6, 5, 4, 3, 2, 1]
+    list3 = list(range(100, 0, -1))
```

Comment:
If you're doing actual comparison to see the timing for the best, worst, and average cases, each list should be the same length